### ISSUE TITLE
New Added Font Family Not Working in annotation. 

### ISSUE THREAD

**Issue by cdpawar08** [ailon/markerjs2#67]
While adding new font family and then add some text on image and save the annotation font family not applied on image.

But in Edit Mode it looks good please check attached image.

![issue-1](https://user-images.githubusercontent.com/16277549/146516658-3f27b7f6-6013-4fbd-888e-9a1e71d4d41a.png)
![issue-2](https://user-images.githubusercontent.com/16277549/146516671-c25c43cc-be6e-4db3-8659-d17c4a0e618f.png)

**Comment by ailon** [comment]
Hi. Thank you for reporting this. To clarify: is this a web font or you just changed the font name to some system font and it doesn't work? If it's a web font, can you please share what it is so I can debug with the exact same setup?

**Comment by cdpawar08** [comment]
Its a web font and below is my code to add this font:

CSS 

```
/* FONT FOR MARKER */
@font-face {
  font-family: 'Avenir LT Pro Medium';
  src: url('./assets/fonts/marker-font/AvenirLTPro-Medium.woff2') format('woff2'),
      url('./assets/fonts/marker-font/AvenirLTPro-Medium.woff') format('woff');
  font-weight: 500;
  font-style: normal;
  font-display: swap;
}

@font-face {
  font-family: 'Avenir LT Pro Light';
  src: url('./assets/fonts/marker-font/AvenirLTPro-Light.woff2') format('woff2'),
      url('./assets/fonts/marker-font/AvenirLTPro-Light.woff') format('woff');
  font-weight: 300;
  font-style: normal;
  font-display: swap;
}

@font-face {
  font-family: 'Avenir LT Pro Book';
  src: url('./assets/fonts/marker-font/AvenirLTPro-Book.woff2') format('woff2'),
      url('./assets/fonts/marker-font/AvenirLTPro-Book.woff') format('woff');
  font-weight: normal;
  font-style: normal;
  font-display: swap;
}
/* FONT FOR MARKER END*/
```

```
markerArea.settings.defaultFontFamilies = [
        "Avenir LT Pro Light",
        "Avenir LT Pro Book",
        "Avenir LT Pro Medium",
      ];
      markerArea.settings.defaultFontFamily = "Avenir LT Pro Book"; 
```

**Comment by cdpawar08** [comment]
Please find Zip for Web Fonts.

[AvenirLTPro.zip](https://github.com/ailon/markerjs2/files/7733919/AvenirLTPro.zip)

**Comment by ailon** [comment]
Yeah, I guess the font needs to be referred in the underlying marker.js SVG directly or otherwise it's not included when rendering. I'll look into it. Thanks!

**Comment by ailon** [comment]
A way to add custom fonts was implemented in v2.21.

Here is the [docs article about this](https://markerjs.com/docs/custom-fonts)
